RGPosadas/LitterRally
# TASK-77: View Other User Profiles Backend

This PR closes #275 

**How To Test**
- Use the addFriend mutation to add rows to the friends pivot table using your favourite GQL playground
- Ensure you have enough users to test the UserRelationship enum (FRIENDS, ADDED_ME, PENDING_REQUEST, and NONE)
